src/styles/index.scss

```diff
@@ -58,11 +58,6 @@ html {
   &:hover {
     cursor: pointer;
   }
-
-  > * {
-    // Prevent click handler from thinking any child elements are the target
-    pointer-events: none;
-  }
```

Comment:
This needs to go to make icons clickable in the graph view.

See https://github.com/cylc/cylc-ui/pull/1780/files#r1589015567